src/test_hash.py

```diff
@@ -0,0 +1,81 @@
+from hash_table import Hash_Table
+import pytest
+import io
+
+KEY = u'key'
+BAD_ITEMS = [1, True, {'Bad': 'Bad'}, ['bad', 13]]
+WORDS = []
+try:
+    wordfile = io.open('src/test_strs.txt', encoding='utf8')
+    WORDS = wordfile.readlines()
+except:
+    pass
+
+
+@pytest.fixture()
+def test_hash():
+    return Hash_Table(10)
+
+
+@pytest.fixture()
+def big_hash():
+    big_hash = Hash_Table(250)
+    for word in WORDS:
+        big_hash.set(word, word)
+    return big_hash
+
+
+def test_hash_init(test_hash):
+    assert len(test_hash._table) == 10
+
+
+def test_hash_hasher(test_hash):
+    assert test_hash._hash(KEY) in range(len(test_hash._table))
+
```

Comment:
Should test that if you try `test_hash.get()` on a key that isn't in your table, it returns a None